pkg/chain/ethereum/ethereum.go

```diff
@@ -490,13 +492,24 @@ func (ec *ethereumChain) CalculateDKGResultHash(
 	return relayChain.DKGResultHashFromBytes(hash)
 }
 
-func (ec *ethereumChain) Address() common.Address {
+func (ec *EthereumChain) Address() common.Address {
 	return ec.accountKey.Address
 }
 
-func (ec *ethereumChain) WeiBalanceOf(address common.Address) (*big.Int, error) {
+func (ec *EthereumChain) WeiBalanceOf(
```

Comment:
We can take the opportunity and make this function an anonymous one inside of `BalanceMonitor()`.

Reply:
Addressed in https://github.com/keep-network/keep-core/commit/89f10b78f353c33ad427d4f7215725039126805b